frontend/public/components/utils/link.tsx

```diff
@@ -85,6 +88,49 @@ export const ExternalLink: React.FC<ExternalLinkProps> = ({
   </a>
 );
 
+// Opens link with copy-to-clipboard
+
+export const ExternalLinkWithCopy: React.FC<ExternalLinkWithCopyProps> = ({
+  link,
+  text,
+  additionalClassName,
+  dataTestID,
+}) => {
+  const [copied, setCopied] = React.useState(false);
+
+  const { t } = useTranslation();
+  const tooltipText = copied
+    ? t('public~Succesfully copied to clipboard')
+    : t('public~Copy to clipboard');
```

Comment:
```suggestion
  const tooltipText = copied
    ? t('public~Copied to clipboard')
    : t('public~Copy to clipboard');
```

This different long text messages create a strangely jumping tooltip when the user presses the copy button:

![copy-after-import](https://user-images.githubusercontent.com/139310/141472553-24843e96-2eca-4b40-ac0e-8d9189f10f0f.gif)

FYI:  I created a PatternFly issue because the tooltip jumps strangly around when you press "copy URL". https://github.com/patternfly/patternfly-react/issues/6589

But I would recommend here to test if this works better with a "similar long" message. (Also if we use this Successfully... message at some other places already.)

Reply:
odd that it jumps.  Thanks for following up with the PF bug!

Reply:
This was already logged long ago: https://github.com/patternfly/patternfly-react/issues/5620